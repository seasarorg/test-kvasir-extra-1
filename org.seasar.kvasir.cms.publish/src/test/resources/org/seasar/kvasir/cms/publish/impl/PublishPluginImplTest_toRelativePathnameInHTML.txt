<html>
  <head>
    <meta http-equiv="Content-Type" content="text/html;charset=UTF-8" />
  </head>
  <a href="/a/c/d.html">THIS IS LINK</a>
  <img src="/a/c/d.html" />
</html>
