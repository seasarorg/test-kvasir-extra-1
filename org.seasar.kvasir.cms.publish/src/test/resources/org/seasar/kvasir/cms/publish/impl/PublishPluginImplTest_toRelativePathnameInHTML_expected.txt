<html>
  <head>
    <meta http-equiv="Content-Type" content="text/html;charset=UTF-8" />
  </head>
  <a href="../c/d.html">THIS IS LINK</a>
  <img src="../c/d.html" />
</html>
